File: 1-TwoSum/1-TwoSum.py
# Last updated: 3/7/2026, 10:14:32 PM
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==target:
                    l.extend([i,j])
                    return l

File: 1-TwoSum/1-TwoSum.java
// Last updated: 3/7/2026, 10:14:30 PM
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n=nums.length;
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if (nums[i]+nums[j]==target){
                    return new int[]{i,j};
                }
            }
        }
        return new int[]{};
    }
}